app/uk/gov/hmrc/gform/email/EmailCodeVerificationController.scala

```diff
@@ -48,11 +48,11 @@ class EmailCodeVerificationController(
               NotifierEmailReference("")
             )
           notifierAlgebra.email(notifierEmail).map(_ => NoContent).toFuture
-        case EmailVerifierService.DigitalContact(emailTemplateId) =>
+        case dc @ EmailVerifierService.DigitalContact(_, _) =>
           emailService
             .sendEmail(
               Some(notifierEmailAddress.value),
-              emailTemplateId.value,
+              dc.emailTemplateId(lang).value,
```

Comment:
Make `sendEmail` to accept `EmailTemplateId` instead of `String`. So that this line can be simplified to 

```
dc.emailTemplateId(lang)
```